### PR TITLE
clear docs

### PR DESCRIPTION
**Pull request by zAG6z**
clear docs, as always change them if you need to

### CODE REVIEW

File: docs/commands/clear.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Administrators Command
+permalink: /docs/commands/administrators/
```
Comment (xf8b) [inline docs/commands/clear.md:4]: this aint the administrators page

File: docs/commands/clear.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+# Clear Command
+##Info
```
Comment (xf8b) [inline docs/commands/clear.md:8]: space

**Review by xf8b — CHANGES_REQUESTED** [review]

File: docs/commands/clear.md (hunk 2 above)
Comment (xf8b) [reply docs/commands/clear.md:8]:
```suggestion
## Info
```

File: docs/commands/clear.md (hunk 1 above)
Comment (xf8b) [reply docs/commands/clear.md]:
```suggestion
title: Clear Command
permalink: /docs/commands/clear/
```

**Comment by xf8b** [comment]
good enough

**Comment by zAG6z** [comment]
alr thx

**Pull request merged**